src/modules/users/dto/upload-doc.dto.ts

```
@@ -0,0 +1,135 @@
+import {
+  IsString,
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)